packages/kbn-securitysolution-io-ts-utils/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@kbn/securitysolution-io-ts-utils",
+  "version": "1.0.0",
+  "description": "elasticsearch datemath parser, used in kibana",
```

Comment:
Change the description here as it says elasticsearch datmath parser and that is wrong.